tap_zendesk/streams.py

```diff
@@ -28,6 +28,9 @@
 
 DEFAULT_SEARCH_WINDOW_SIZE = (60 * 60 * 24) * 30 # defined in seconds, default to a month (30 days)
 
+def get_sideload_objects(stream):
+    return metadata.to_map(stream.metadata).get((), {}).get('sideload-objects')
```

Comment:
Can you give small details on what this will return if there are no sideload objects? Maybe in the function's docstring?

Reply:
added a line to explain the method